server/database/models/cartItems.js

```diff
@@ -0,0 +1,11 @@
+const Sequelize = require("sequelize");
+const db = require("../connection");
+
+const CartItems = db.define("cartitems", {
+  quantity: {
```

Comment:
There are definitely some better approaches to quantity.

Reply:
One interesting thought straight from Sequelizes [_always beautiful docs_](https://sequelize.readthedocs.io/en/2.0/docs/associations/) is:

> If you want additional attributes in your join table, you can define a model for the join table in sequelize, before you define the association, and then tell sequelize that it should use that model for joining, instead of creating a new one:

```javascript
User = sequelize.define('User', {})
Project = sequelize.define('Project', {})
UserProjects = sequelize.define('UserProjects', {
    status: DataTypes.STRING
})
 
User.belongsToMany(Project, { through: UserProjects })
Project.belongsToMany(User, { through: UserProjects })
```